webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -321,17 +321,26 @@ private static PdfReader createPdf(String globalTitle, Date creationDate, Scramb
                     Element.ALIGN_CENTER, new Phrase(scrambleRequest.title),
                     (pageSize.getLeft() + pageSize.getRight()) / 2, pageSize.getTop() - 45, 0);
 
-            if(pr.getNumberOfPages() > 1) {
-                ColumnText.showTextAligned(cb,
-                        Element.ALIGN_RIGHT, new Phrase(pageN + "/" + pr.getNumberOfPages()),
-                        rect.getRight(), rect.getTop(), 0);
-            }
+            //if(pr.getNumberOfPages() > 1) {
+            //    ColumnText.showTextAligned(cb,
+            //            Element.ALIGN_RIGHT, new Phrase(pageN + "/" + pr.getNumberOfPages()),
+            //            rect.getRight(), rect.getTop(), 0);
+            //}
 
             // Footer
-            String generatedBy = "Generated by " + Utils.getProjectName() + "-" + Utils.getVersion();
-            ColumnText.showTextAligned(cb,
-                    Element.ALIGN_CENTER, new Phrase(generatedBy),
-                    (pageSize.getLeft() + pageSize.getRight()) / 2, pageSize.getBottom() + 40, 0);
+			switch (pr.getNumberOfPages % 2) {
```

Comment:
I'm interested in learning more about what you're doing here, but can you do this in a separate PR? Is this related to the Vietnamese translation somehow?

Reply:
I changed the way the number of pages are being printed. But this is for myself only. I thought the pull request only contains my Vietnamese translation... :/ Please don't notice my changes to the code.